code/src/components/AlbumList/AlbumList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Album } from '../Album/Album';
+
+export const AlbumList = (props) => {
+  console.log(props)
+  return (
+    <section className="AlbumList">
```

Comment:
It seems that it's only the artist that is a link. It would be cool if you also could click the album-name and then you're brought to the album. I think you can accomplish this the same way you fetched the link for the artist. :)